### PR TITLE
move tests to _test package

### LINKED ISSUE #2359: Use the “underscore test” for the handler package
**Description**
Go doesn’t let you include multiple packages in one folder except for when you use a separate test package.

Using a separate `_test` package means that user can’t access the unexported fields and functions in the package. The test can see only exported API, so it makes the code more usable, safe and complete.

### PR DESCRIPTION
**Pull request by zreigz**
**What this PR does / why we need it**:  this is the last step to moving unit tests from handler package to handler_test package

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #2359

```release-note
NONE
```

### CODE REVIEW

File: api/pkg/handler/routing_test.go [hunk 1]
```diff
@@ -1,116 +1,29 @@
...
 	testUserID    = test.UserID
 	testUserName  = test.UserName
 	testUserEmail = test.UserEmail
 )
 
-func getUser(email, id, name string, admin bool) apiv1.User {
-	return test.GetUser(email, id, name, admin)
-}
-
 func TestUpRoute(t *testing.T) {
```
Comment (p0lyn0mial) [inline api/pkg/handler/routing_test.go]: does this test has any real value ? It test an endpoint that always returns OK. How do you feel about removing it ?

Comment (zreigz) [reply api/pkg/handler/routing_test.go]: agree

Comment (zreigz) [reply api/pkg/handler/routing_test.go]: I removed the file

**Comment by p0lyn0mial** [comment]
fyi the tests are failing because there are no sufficient resources, you might want to wait until Monday :) (Henrik is working on it)

**Pull request merged**